src/sinol_make/commands/export/__init__.py

```diff
@@ -0,0 +1,287 @@
+import os
+import glob
+import shutil
+import tarfile
+import tempfile
+import argparse
+import yaml
+
+from sinol_make import util
+from sinol_make.helpers import package_util, parsers
+from sinol_make.commands.gen import gen_util
+from sinol_make.interfaces.BaseCommand import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Class for "export" command.
+    """
+
+    def get_name(self):
+        return "export"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Create archive for oioioi upload',
+            description='Creates archive ready to upload to oioioi. '
+                        'It is possible to create ocen and dlazaw debian packages for usage in contests '
+                        'with --ocen flag.'
+        )
+
+        parser.add_argument('-o', '--output', type=str, default='export',
+                            help='output directory. Default: export')
+        parser.add_argument('--ocen', action='store_true', default=False,
+                            help='create ocen (or dlazaw in case `dlazaw/` directory exists) debian package '
+                                 'for usage in contests')
+        parsers.add_compilation_arguments(parser)
+
+    def get_generated_tests(self):
+        """
+        Returns list of generated tests.
+        Executes ingen to check what tests are generated.
+        """
+        with tempfile.TemporaryDirectory() as tmpdir:
+            ingen_path = gen_util.get_ingen(self.task_id)
+            if os.path.splitext(ingen_path)[1] == '.sh':
+                ingen_exe = ingen_path
+            else:
+                ingen_exe = gen_util.compile_ingen(ingen_path, self.args, self.args.weak_compilation_flags)
+            if not gen_util.run_ingen(ingen_exe, tmpdir):
+                util.exit_with_error('Failed to run ingen.')
+
+            tests = glob.glob(os.path.join(tmpdir, f'{self.task_id}*.in'))
+            return [package_util.extract_test_id(test) for test in tests]
```

Comment:
So, I wonder, do we have to run the ingen in extract? While it's true that the old Makefile scripts did so, maybe there is a better way. Why not just taking all `.in` files? Is it a good idea, or a bad idea?

Reply:
There is a size limit for uploading packages, so I think it’s quite important

Reply:
Ah, you mean to *not* put tests that are generated?

Reply:
Yes

Reply:
Hmm, what about requiring from the user to automatically generate each input file? Then also regenerating tests would be easier, as we can just remove each test.

Reply:
But what about a task with multiple correct answers to one input and in the task statement there is an explanation to a hand written output? It would also break many existing packages as there are some that don't even have ingen.

Reply:
> But what about a task with multiple correct answers to one input and in the task statement there is an explanation to a hand written output? 

An usual solution is to make an `if` in the model solution for this specific test and print the wanted output.

> It would also break many existing packages as there are some that don't even have ingen.

Fair enough. Let's stick with that. Though I think it should print a warning for any tests it copies that is hand-written, as maybe it is a test left out and it shouldn't be there.

Reply:
I added a warning for this